packages/snowpack/src/build/build-pipeline.ts

```diff
@@ -50,7 +50,7 @@ async function runPipelineLoadStep(
         messageBus.emit(msg, {
           ...data,
           id: step.name,
-          msg: data.msg && `[${srcPath}] ${data.msg}`,
+          msg: data.msg && `${data.msg} (${srcPath})`,
```

Comment:
Minor change: message first, path 2nd. I found this to be a little more readable.